src/test/excessiveblock_test.cpp

```diff
@@ -229,6 +238,9 @@ BOOST_AUTO_TEST_CASE(check_generated_block_validator)
 
     str = MiningBlockSizeValidator(tmpMGB, (uint64_t *) 42, true);
     BOOST_CHECK(! str.empty());
+
+    maxGeneratedBlock = c_mgb;
+    excessiveBlockSize = c_ebs;
 }
```

Comment:
This was a bug in my unit test code. The tests themselves rely on the state of the system. I did not see that. Thus (and I assume) when the tests are run by travis they are order randomised. Thus my test *sometimes* caused issues for other tests if run out of order by changing the state. Thus I think this change is where it belongs as part of this pull request.